Doc/library/os.rst

```diff
@@ -715,6 +715,29 @@ as internal buffering of data.
               pass
 
 
+.. function:: copy_file_range(src, dst, count, offset_src=None, offset_dst=None)
+
+   Copy *count* bytes from file descriptor *src*, starting from offset
+   *offset_src*, to file descriptor *dst*, starting from offset *offset_dst*.
+   If *offset_src* is None, then *src* is read from the current position;
+   respectively for *offset_dst*. The files pointed by *src* and *dst*
+   must reside in the same filesystem, otherwise an :exc:`OSError` is
+   raised with :attr:`~OSError.errno` set to :data:`errno.EXDEV`.
+
+   This copy is done without the additional cost of transferring data
+   from the kernel to user space and then back into the kernel. Additionally,
+   some filesystems could implement extra optimizations. The copy is done as if
+   both files are opened as binary.
+
+   The return value is the amount of bytes copied. This could be less than the
+   amount requested.
+
+   Availability: Linux 4.5 (glibc 2.27 provides a user-space emulation when it
```

Comment:
Should probably make it explicit that the mentioned versions of linux and glibc are the first versions where this support was added, e.g. `Availability: Linux 4.5+ (glibc 2.27+ provides ...)`

Reply:
Fixed in 6c6cce6